pkg/driver/node/mounter/pod_mounter_linux.go

```diff
@@ -47,6 +47,8 @@ func (pm *PodMounter) mountSyscallDefault(target string, args mountpoint.Args) (
 
 	if args.Has(mountpoint.ArgReadOnly) {
 		flags |= syscall.MS_RDONLY
+		// Remove the read-only argument from the list as mount-s3 does not support it when using FUSE file descriptor
+		args.Remove(mountpoint.ArgReadOnly)
```

Comment:
Should we move this to `PodMounter.Mount` function after calling this method? Seems like it would be better there rather than mutating the `args`. For example the caller might pass you a copy of `args` and this would stop working

Reply:
Good point! Moved to `PodMounter.Mount`